docker-compose.yml

```diff
@@ -1,15 +1,80 @@
 version: "3.1"
 
 services:
+  conseil:
+    image: cryptonomictech/conseil:0.2002.0020-SNAPSHOT
```

Comment:
yes, if you look at the postgres image it is also hardcoded to `11.6` 😉 we just have it in the different format(based on `BuildInfo.scala` version)

Reply:
What I mean is that we're now bound to manually update those values to the latest stable version, at every official release, both here and in the lorre section?

Why not some dynamic value or `latest`?

Reply:
about latest tag: https://vsupalov.com/docker-latest-tag/ - it does not work how you probably think it works - been there, done that 😄
`latest` is being called anti-pattern most of the time.
I don't believe we have to change it every release.  What's wrong with user checking docker-hub for the tag of the newest version and replacing it manually? Because I don't think we are going to use it for the development.

Reply:
Ok, so the idea is that we provide this for the user, not for the developers (as hinted by the current root readme file).
That was the original intention, and I'm ok with changing that as long as we document the purpose correctly to anyone that is approaching the project from github

Reply:
I was also thinking about providing this as a separate docker-compose file and leave current as it is, WDYT?